dev.yml

```diff
@@ -1,6 +1,8 @@
 name: ejson
 
 up:
+  - homebrew:
+    - gnu-tar
```

Comment:
Is it enough to add the package? I know that `gnu-tar` adds `gtar` but doesn't override the `tar` distributed by macOS

Reply:
It did work out of the box for me.

Reply:
I had this same thought but if it works, 🤷‍♂